boards/nrf52dk/Makefile.include

```diff
@@ -10,6 +10,7 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
 export JLINK_DEVICE := nrf52
 
 # special options when using SoftDevice
+include $(RIOTBOARD)/$(BOARD)/Makefile.dep
```

Comment:
As this is a workaround, I guess it can be valid. However, I think we need to put a note here stating that **this is incorrect**. We shouldn't include the dependencies makefile here, but only in another Makefile.dep. @cladmi might have a proposition about the needed comment.

I noticed this is also done for some arduino boards, I'll check why it's there and if we can avoid it.

Reply:
Then I'll wait for your inputs for this point :)